wsmaster/che-core-api-factory/src/main/java/org/eclipse/che/api/factory/server/urlfactory/DefaultFactoryUrl.java

```diff
@@ -40,6 +41,18 @@ public String location() {
         });
   }
 
+  // never really used, just for compatibility
```

Comment:
shouldn't it be abstract then? or throw an exception? or return null?

Reply:
i would left it, as it is properly implemented, just not used now (but someone can use it). So i;''ll just remove those confusing comments. Ok?